packages/trader/src/Modules/MT5/Containers/mt5-password-modal.jsx

```diff
@@ -285,6 +286,7 @@ const MT5PasswordModal = ({
         disableMt5PasswordModal();
         closeDialogs();
         if (account_type.category === 'real') {
+            localStorage.setItem('mt5_transfer_to_login_id', mt5_new_account.login);
```

Comment:
Do you think we can use sessionStorage for this item instead in case we don't need to keep the value till the next session?

Reply:
yes, sessionStorage should be preferred here. Thanks for the suggestion 🙇